docs/source/components/nodes/stereo_depth.rst

```diff
@@ -375,8 +375,18 @@ or roughly 35cm.
 
 See `these examples <https://github.com/luxonis/depthai-experiments/tree/master/gen2-camera-demo#real-time-depth-from-depthai-stereo-pair>`__ for how to enable Extended Disparity.
 
+Disparity shift to lower min depth perception
+---------------------------------------------
+
+Another option to perceive closer depth range is to use disparity shift. Disparity shift will shift the starting point
```

Comment:
Add some graphs. You can take them from realsense docs.
Same for a more detailed description:
https://github.com/luxonis/depthai-python/pull/615